docs/installation_guide.rst

```diff
@@ -359,7 +359,6 @@ rotki uses an encrypted database called `SQLCipher <https://www.zetetic.net/sqlc
 
     $ brew update
     $ cd "$(brew --repo homebrew/core)"
-    $ git checkout 2731bfdc785e85de8242b164acbf4fcb627a91e2 Formula/sqlcipher.rb #This formula installs 4.4.0 of sqlcipher
```

Comment:
Can you actually change it to 4.4.3 as you said and not remove it?

The docs here are outdated. 4.4.3 is also what we use in the packaging script for OSX: https://github.com/rotki/rotki/blob/1950e23d3fca491987430f35e0324fb5410e1f2e/.github/workflows/rotki_packaging.yaml#L213-L218